packages/cli/src/requests.ts

```diff
@@ -386,3 +386,19 @@ export type BinaryDataRequest = AuthenticatedRequest<
 		mimeType?: string;
 	}
 >;
+
+// ----------------------------------
+//              /sso
+// ----------------------------------
+
+export declare namespace SSO {
+	export type AcsRequest = AuthenticatedRequest<
+		{},
+		{},
+		{
+			// eslint-disable-next-line @typescript-eslint/naming-convention
+			RelayState?: string;
```

Comment:
since this is an internal field, can we rename it to `relayState` to remove the linting exception, and to be more consistent with body-fields from other request types in the codebase.

@flipswitchingmonkey would renaming this be an issue?

Reply:
@netroy the field comes in as RelayState (capitalized) by SAML convention from the Identiy Provider.